src/xunit.console.netcore/CommandLine.cs

```diff
@@ -14,18 +15,59 @@ protected CommandLine(string[] args, Predicate<string> fileExists = null)
                 fileExists = fileName => File.Exists(fileName);
 
             for (var i = args.Length - 1; i >= 0; i--)
+            {
+                if (args[i][0] == '@')
+                {
+                    // Parse response file
+                    IList<string> rspArguments = ParseResponseFile(args[i].Substring(1));
+                    for (int j = rspArguments.Count - 1; j >= 0; j--)
+                        arguments.Push(rspArguments[j]);
+                    continue;
+                }
                 arguments.Push(args[i]);
+            }
 
             TeamCity = Environment.GetEnvironmentVariable("TEAMCITY_PROJECT_NAME") != null;
             AppVeyor = Environment.GetEnvironmentVariable("APPVEYOR_API_URL") != null;
             Project = Parse(fileExists);
         }
 
+        /// <summary>
+        /// Parse a response file passed as a command-line arguments.
+        /// No verification here to make this completely opaque
+        /// </summary>
+        /// <param name="responseFile">Path to the response file</param>
+        /// <param name="arguments">The data structure in</param>
+        private IList<string> ParseResponseFile(string responseFile, char[] delimiters = null)
+        {
+            // Accept formats of both -Qualifier Name and -Qualifier:Name
+            if (delimiters == null)
+                delimiters = new char[] { ' ', ':' };
```

Comment:
Why the ":' delimiter?

Reply:
It's probably not strictly necessary. I copied the support for the format from 

https://github.com/dotnet/buildtools/blob/8d796b96a0199b0207ae54cb45e0d12c6fa7edf0/src/common/CommandLine.cs#L25-L29

The syntax Qualifier=Name is already supported in the xunit runner, so I added this to keep in line with other .rsp file formats.

Reply:
I'd stay consistent with the other arg parsing in this file. I don't see ":" or "=" supported it appears as only " " is supported so I would do the same here.